docs/SUMMARY.md

```diff
@@ -1,15 +1,18 @@
 # Table of contents
 
-* [Aragon Govern](README.md)
+* [Introduction](README.md)
 
-## High level overview <a id="guides"></a>
+## Overview <a id="guides"></a>
```

Comment:
I think using “Overview” is enough and aligns more with the other section name (“Developers”).

Reply:
Fully agree, it's way better!